src/index.html

```diff
@@ -12,7 +12,39 @@
       href="styles/index.scss"
```

Comment:
The href attribute is pointing to a `.scss` file. Browsers cannot directly interpret SASS files. Ensure that this file is being compiled into a CSS file, and update the href to point to the compiled CSS file instead.